project-base/tests/App/Acceptance/acceptance/OrderCest.php

```diff
@@ -33,22 +42,23 @@ public function testFormRemembersPaymentAndTransportWhenClickingBack(
 
         // tv-audio
         $me->amOnLocalizedRoute('front_product_list', ['id' => 3]);
-        $productListPage->addProductToCartByName('Defender 2.0 SPK-480');
-        $me->waitForAjax();
+        $productListPage->addProductToCartByName(self::DEFAULT_PRODUCT_NAME);
+
         $orderPage->clickGoToCartInPopUpWindow();
-        $me->clickByTranslationFrontend('Order [verb]');
 
-        $orderPage->assertTransportIsNotSelected('Czech post');
+        $orderPage->continueToSecondStep();
+
+        $orderPage->assertTransportIsNotSelected(self::DEFAULT_TRANSPORT_NAME);
```

Comment:
_nitpick_ 🤓 

shouldn't we check if no transport is selected instead of the default one? (same for payment)
I'm ok with leaving as it is as it's a current behavior

Reply:
I think it is ok as it is